src/client/Client.cc

```diff
@@ -6136,7 +6136,7 @@ void Client::_abort_mds_sessions(int err)
 
 void Client::_unmount(bool abort)
```

Comment:
This is also called by `C_Deleg_Timeout::finish`. This refactor doesn't look possible unfortunately.

See: /ceph/teuthology-archive/pdonnell-2020-07-21_06:19:42-fs-wip-pdonnell-testing-20200721.040813-distro-basic-smithi/5246359/teuthology.log

Reply:
Fixed it, thanks